app/jobs/bulk_actions_job.rb

```diff
@@ -0,0 +1,57 @@
+class BulkActionsJob < ApplicationJob
+  queue_as :medium
+  attr_accessor :records
+
+  MODEL_TYPE = ['Conversation'].freeze
+
+  def perform(account:, params:, user:)
+    @account = account
+    Current.user = user
```

Comment:
@tejaswinichile  just noticed this. If we set current.user here. it will remain in the thread. So we need to ensure that it's unset at the end of the execution. We also need to vary of cases where the job could fail and the unsettling of Current.user doesn't happen. 

Probably we can have an initializer in the application job which unsets the Current. user as the start of the execution of each job to sure.

Reply:
@sojan-official  Yes, this is for the activity message, let me unset it after the execution.